package.json

```diff
@@ -53,6 +53,11 @@
     ]
   },
   "devDependencies": {
-    "tailwindcss": "^3.4.1"
+    "tailwindcss": "^3.4.1",
```

Comment:
Not something you should fix, but just an article to read:
[What is `~`, `^`](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) and [some examples](https://semver.npmjs.com/#syntax-examples)

Reply:
Okay, thanks